test/e2e/data/metadata/cluster-api/metadata.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 kind: Metadata
 releaseSeries:
+  - major: 1
```

Comment:
Need to do the same change in https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean/blob/main/metadata.yaml

Reply:
Sounds good! I've added it [here](https://github.com/kubernetes-sigs/cluster-api-provider-digitalocean/pull/509/files#diff-d186204ea7abb85bbadd9d94ab29d5bcefcc43d6a0ed06015e98a71b2f099423R22).